docs/06.REFERENCE-COUNTING.md

```diff
@@ -76,20 +77,21 @@ shows this behaviour through property getting and setting.
 
   /* Property setting is the same. */
 
+  jerry_value_t new_prop_value = jerry_create_number (2.718);
```

Comment:
I thought this made it more clear to show where new_prop_value comes from, like in all the API doc samples.

If it was passed in from another C function, it might *not* need to be released here, but in the caller. So this shows it was acquired right here and needs to be released here. If you agree, maybe the same thing should be added to the "simplest form" snippet right below this example too, but I didn't do that.